app/services/post_persister_task_service.rb

```diff
@@ -14,6 +14,7 @@ def validate_options
   def update_source
     source = Source.find(@options[:task].source_id)
     source.update!(:last_successful_refresh_at => @options[:task][:input]["refresh_request_at"]) if @options[:task].status == "ok"
-    source.update!(:refresh_finished_at => Time.current)
+    source.update!(:refresh_finished_at => Time.current, :refresh_state => "Done")
```

Comment:
@hsong-rh Should this set the refresh_state to error if the task fails. We would have to look at the Task#status field